store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9629,4 +9629,13 @@ TODO: delete them permanently from here
     <desc>Account thumbnail photo</desc>
 </attr>
 
+<attr id="3022" name="zimbraSmtpStartTlsMode" type="enum" value="on,off,only" cardinality="single" optionalIn="globalConfig,server,domain" flags="serverInherited" since="8.8.6">
```

Comment:
Is this really needed on server and domain both?
It can be optionalIn="globalConfig,server" flags="serverInherited" or 
optionalIn="globalConfig,domain" flags="domainInherited"

Reply:
This attribute is relevant to zimbraSmtpHostname, zimbraSmtpPort, and zimbraSmtpTimeout.
These are all defined with optionalIn="globalConfig,server,domain".
I think we should better to keep it consistent.